pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -2200,6 +2200,7 @@ type BackupScheduleSpec struct {
 	// BackupTemplate is the specification of the backup structure to get scheduled.
 	BackupTemplate BackupSpec `json:"backupTemplate"`
 	// LogBackupTemplate is the specification of the log backup structure to get scheduled.
+	// +optional
```

Comment:
only one line change, this field was already optional during implementation, it's just that the comment wasn't written correctly when generating the CRD.
see more in  https://github.com/pingcap/tidb-operator/blob/e2eb643abb916684c1dfcb420441561300ddf2c0/pkg/backup/backupschedule/backup_schedule_manager.go#L138

Reply:
ci failed when checking forward compatibility, maybe we should force merge this pr.

https://github.com/pingcap/tidb-operator/actions/runs/4605029707/jobs/8136576127?pr=4956